vowpalwabbit/namespaced_feature_store.h

```diff
@@ -4,253 +4,217 @@
 
 #pragma once
 
-#include <utility>
-#include <vector>
-#include <unordered_map>
 #include <cassert>
-#include <set>
+#include <list>
+#include <array>
+#include <vector>
 
 #include "feature_group.h"
-#include "generic_range.h"
 #include "chained_proxy_iterator.h"
 
 typedef unsigned char namespace_index;
```

Comment:
sort-of unrelated to PR but can this be a `using namespace_index` instead of `typedef`?

Reply:
Yeah this is inconsistent, all new places i use `using`. I agree with you